applications/facings-nft-collection-manager.md

```diff
@@ -0,0 +1,233 @@
+
+
+## Development Roadmap
+
+
+### Milestone Summary
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):** 4 FTE
+- **Total Costs:** 80,000 USD
+
+
+### Milestone 1 — Account Login & View Collections
+
+- **Estimated duration:** 1 month
+- **FTE:**  4
+- **Costs:** 40,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | GPLv3 |
+| 0b. | Documentation | We will provide a ‘Getting Started’ guide, deployment instructions, as well as inline documentation of the code. |
+| 0c. | Testing Guide | Documentation for a walkthrough of the UI as described and the intended outcome of each subset will be provided along with expected feedback results. FACINGS UI standards including visibility of system status, user control and freedom, consistency, error prevention, recognition over recall, flexibility and efficiency, minimalistic design, error recovery and documentation will all be implemented. |
+| 0d. | Docker | We will provide a Dockerfile that can be used to launch a UI instance and test all the functionality delivered with this milestone. |
+| 1. | Base Implementation | Home page: show featured collections, “about” section, and login link. |
+| 2. | Login & Account | When logged in, show: (a) logged in user’s account name and profile photo, (b) Option to log out, (c) RAM/Resource usage, and (d) list of user's collections. |
+| 3. | Collection viewer | Ability to explore/examine Collections, Schemas, Templates, Assets. |  
+
+
+
+### Milestone 2 — Create/Edit Collections
+
+- **Estimated Duration:** 1 month
+- **FTE:**  4
+- **Costs:** 40,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | GPLv3 |
+| 0b. | Documentation | We will provide a ‘Getting Started’ guide, deployment instructions, as well as inline documentation of the code. |
+| 0c. | Testing Guide | Documentation for a walkthrough of the UI as described and the intended outcome of each subset will be provided along with expected feedback results. FACINGS UI standards including visibility of system status, user control and freedom, consistency, error prevention, recognition over recall, flexibility and efficiency, minimalistic design, error recovery and documentation will all be implemented. |
+| 0d. | Docker | We will provide a Dockerfile that can be used to launch a UI instance and test all the functionality delivered with this milestone. |
+| 0e. | Article | We will publish an article describing the work we performed as part of the grant, introduce our app, and highlight the features which make it valuable. |
+| 1. | Collection Manager | Handle fundamental operations: (1) Create/Edit Collections, (2) Create/Extend Schemas, (3) Create Templates, (4) Create Assets |  
```

Comment:
is `(4) Create Assets` synonymous with performing an Airdrop? 

In the project details section it is implied that after Phase 1's complete delivery users would be able to at least perform airdrop-style distribution of NFTs.  If (4) is _not_ that where will that functionality be accounted for?

Reply:
“Create Assets” is referring to UI controls which enable you to specify how many assets to mint from a template, and which account to mint them into. We consider Airdrops as mentioned in Stage 1 a minimal workflow to start, but which enables a list of accounts to be provided for distribution, to which an NFT will be minted from a template, or distributed to an account via a transfer if the asset is already minted.

I’m adding  “(5) Airdrop assets to list of accounts” as described above to the 2nd milestone, thanks for noticing that we forgot it there.